### PR TITLE
Add Python 3.6 to Continuous Integration

### PR DESCRIPTION
**Pull request by astronouth7303**
Seems to be some oddities with Appveyor, though. Help?

Internal. No news.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,7 +1,19 @@
 version: 0.5.2.{build}
 os: Windows Server 2012 R2
+environment:
+
+  matrix:
+    # http://www.appveyor.com/docs/installed-software#python
+    - PYTHON: "C:\\Python34"
```
Comment (scopatz) [inline .appveyor.yml]: We can probably stop testing on Python v3.4

**Comment by scopatz** [comment]
Theee seem to be legitimate errors on Python v3.6 on Windows. It looks like some of the context management stuff might not work yet.

File: .appveyor.yml (hunk 1 above)
Comment (astronouth7303) [inline .appveyor.yml]: Probably? Although I know it's the version installed by default on Amazon Linux. (Aka, what's default on EC2.)

**Comment by astronouth7303** [comment]
That's pretty weird that something that doesn't involved system interaction is only broken on Windows and 3.6.

File: .appveyor.yml (hunk 1 above)
Comment (laerus) [inline .appveyor.yml]: debian-stable will still be python3.4 for some time too.

Comment (scopatz) [inline .appveyor.yml]: That is fair. I guess we can keep it around

**Comment by scopatz** [comment]
Potential fix in #2074

**Pull request merged**